Allow scroll restoration to be disabled (#1431)
Shopify/hydrogen: docs/components/framework/link.md

{% endcodeblock %}

## Scroll restoration

> By default, when a `<Link>` component is clicked, Hydrogen emulates default browser behavior and attempts to restore the scroll position previously used in the visitor's session. For new pages, this defaults to scrolling to the top of the page.

Reviewer: ```suggestion
By default, when you click a `<Link>` component, Hydrogen emulates default browser behavior and attempts to restore the scroll position that was previously used in the visitor's session. For new pages, the  `<Link>` component defaults to scrolling to the top of the page.
```

> However, if you are building a user interface that should fetch a new server components request and update the URL but not modify scroll position, then you can disable scroll restoration using the `restoreScroll` prop:

Reviewer: ```suggestion
However, if you want to build a user interface that fetches a new server component's request and updates the URL, but doesn't modify the scroll position, then you can disable scroll restoration using the `restoreScroll` prop:
```

Reviewer 2: @mcvinci I suggest this instead:

> However, if you want to build a user interface that re-renders server components and updates the URL, but doesn't modify the scroll position, then you can disable scroll restoration using the `restoreScroll` prop:

Reviewer: Love it! Thanks, @blittle!
